robyn/ws.py

```diff
@@ -35,3 +38,26 @@ def _num_params(self, handler):
 
     def _is_async(self, handler):
         return asyncio.iscoroutinefunction(handler)
+
+    def send(self, message: str) -> None:
+        print("This is the url ", self.url)
+        # making new connection each time
+        # is very slow
+        # expose sync_connect outside of this class
+        with sync_connect(self.url) as websocket:
+            websocket.send(message)
+
+    def recv(self):
+        with sync_connect(self.url) as websocket:
```

Comment:
the context manager should open the connection on the execution of `@app.ws("/")`